src/Generator.php

```diff
@@ -27,12 +29,9 @@ class Generator
      * @param int $minimumLength
      * @return string
      */
-    public function calcMod10V1($number, $separator = "", $minimumLength = 6)
+    public function calcMod10V1(string $number, string $separator = "", int $minimumLength = 6) : string
```

Comment:
There shouldn't be a space between the closing parenthesis and the colon.

Reply:
Resolved.